bitnami/nats/templates/networkpolicy.yaml

```diff
@@ -8,7 +8,7 @@ kind: NetworkPolicy
 apiVersion: {{ include "common.capabilities.networkPolicy.apiVersion" . }}
 metadata:
   name: {{ template "common.names.fullname" . }}
```

Comment:
I found in this file an incorrect reference: Values.rts.networkPolicy. Could you please check it?

Reply:
done at https://github.com/bitnami/charts/pull/31516/commits/563a4df661a5e8cdfd53908f3b2a37b2b8c3af16